utils/src/main/java/se/fortnox/reactivewizard/util/rx/RetryWithDelay.java

```diff
@@ -1,48 +1,44 @@
 package se.fortnox.reactivewizard.util.rx;
```

Comment:
I like Project Reactor -> Projector :D 
These changes will make it unusable for rxjava and for projector it seems pretty unnecessary. I suggest we either remove it or keep it until rxjava support is gone.

Reply:
Agreed! Closing PR.